pkgs/development/python-modules/behave/default.nix

```diff
@@ -1,21 +1,21 @@
 { stdenv, fetchPypi
-, buildPythonApplication, python, pythonAtLeast
-, mock, nose, pyhamcrest
+, buildPythonApplication, isPy27, python
+, mock, nose, pathpy, pyhamcrest, pytest
 , glibcLocales, parse, parse-type, six
 }:
 buildPythonApplication rec {
   pname = "behave";
-  version = "1.2.5";
+  version = "1.2.6";
   name = "${pname}-${version}";
 
-  disabled = pythonAtLeast "3.6";
+  disabled = isPy27;
```

Comment:
So this is not supposed to be executed with the python version one wants to run the tests with? https://github.com/behave/behave

Reply:
You are correct and I'll add python27 back in.  Thanks for pointing out a bad solution.

On a slightly tangential note, how much longer will python27 be the default python installation and available on NixOS?